src/google/protobuf/message_lite.cc

```diff
@@ -186,7 +186,7 @@ class ZeroCopyCodedInputStream : public io::ZeroCopyInputStream {
   }
   void BackUp(int count) final { cis_->Advance(-count); }
   bool Skip(int count) final { return cis_->Skip(count); }
-  int64 ByteCount() const final { return 0; }
+  google::protobuf::io::ByteCountInt64 ByteCount() const final { return 0; }
```

Comment:
I think this should be int64_t. Can you please check with Josh?

Reply:
Fixed internally